docs/integrations.rst

```diff
@@ -238,12 +238,31 @@ Snyk
 ----
 Snyk output file (snyk test --json > snyk.json) can be imported in JSON format.
 
-SonarQube
+SonarQube Scan (Aggregates findings per cwe, title, description, file_path.)
 ---------
 SonarQube output file can be imported in HTML format.
 
 To generate the report, see https://github.com/soprasteria/sonar-report
 
+Version: >= 1.1.0
+
+SonarQube Scan Detailed (mport all findings from sonarqube html report.)
+---------
+SonarQube output file can be imported in HTML format.
+
+To generate the report, see https://github.com/soprasteria/sonar-report
+
+Version: >= 1.1.0
+
+SonarQube API Import
+---------

```

Comment:
Make the hyphens extend to the end of the line of text above them. It's a syntax error otherwise.

I've not used the.  SonarQube API, so not sure on the content.

Reply:
I have verified Sonarqube API and have been using this currently and thats the reason i decided to update the document

Reply:
Hypens have been updated now